docs/content.zh/docs/ops/debugging/profiler.md

```diff
@@ -25,57 +25,57 @@ specific language governing permissions and limitations
-- **ITIMER**: You can fall back to itimer profiling mode. It is similar to cpu mode, but does not require perf_events support. As a drawback, there will be no kernel stack traces.
+- **CPU**: 在这种模式下，分析器收集包括Java方法、本地调用、JVM代码和内核函数在内的堆栈跟踪样本
+- **ALLOCATION**: 在allocation 分析模式下， 每个调用跟踪的顶部框架是被分配对象的类，计数器是堆压力 （分配的TLAB或TLAB之外的对象的总大小）。
+- **Wall-clock**: Wall-Clock选项使得异步分析器在每个给定的时间段内均匀对所有线程进行采样，无论线程状态如何：正在运行、睡眠或阻塞。这个选项在分析应用程序启动时间时会很有帮助
+- **Lock**: 在lock分析模式下，顶部框架是锁定/监视器的类，计数器是进入此锁定/监视器所需的纳秒数。
+- **ITIMER**: 您可以退回到itimer分析模式。它类似于cpu模式，但不需要perf_events支持。它的缺陷是没有内核堆栈跟踪。
 
 {{< hint warning >}}
 
-Any measurement process in and of itself inevitably affects the subject of measurement. In order to prevent unintended impacts on production environments, Profiler are currently available as an opt-in feature. To enable it, you'll need to set [`rest.profiling.enabled: true`]({{< ref "docs/deployment/config">}}#rest-profiling-enabled) in [Flink configuration file]({{< ref "docs/deployment/config#flink-configuration-file" >}}). We recommend enabling it in development and pre-production environments, but you should treat it as an experimental feature in production.
+任何测量过程本身都不可避免地会影响测量对象。为了防止对生产环境产生意外影响，分析器目前作为一项可选的功能。要启用它，您需要在[Flink配置文件]({{< ref "docs/deployment/config#flink-configuration-file" >}}) 中设置 [`rest.profiling.enabled: true`]({{< ref "docs/deployment/config">}}#rest-profiling-enabled) 。我们建议在开发和预生产环境中启用它，但在生产环境中应将其视为试验性功能。
 
 {{< /hint >}}
 
 
-## Requirements
-Since the Profiler is powered by the Async-profiler, it is required to work on platforms that are supported by the Async-profiler.
+## 要求
+由于分析器由异步分析器驱动，因此必须在异步分析器支持的平台上运行。
 
 |           | Officially maintained builds | Other available ports                     |
 |-----------|------------------------------|-------------------------------------------|
 | **Linux** | x64, arm64                   | x86, arm32, ppc64le, riscv64, loongarch64 |
 | **macOS** | x64, arm64                   |                                           |
 
-Profiling on platforms beyond those listed above will fail with an error message in the `Message` column.
+在上述之外的平台分析将在 `Message` 列中报错。
 
 
-##  Usage
-Flink users can complete the profiling submission and result export via Flink Web UI conveniently.
+##  用途
+Flink用户可以通过Flink Web UI方便地完成剖析提交和结果导出。
 
-For example,
-- First, you should find out the candidate TaskManager/JobManager with performance bottleneck for profiling, and switch to the corresponding TaskManager/JobManager page (profiler tab).
-- You can submit a profiling instance with a specified duration and mode by simply clicking on the button **Create Profiling Instance**. (The description of the profiling mode will be shown when hovering over the corresponding mode.)
-- Once the profiling instance is complete, you can easily download the interactive HTML file by clicking on the link.
+比如，
+- 首先，您应找出性能瓶颈的 TaskManager/JobManager，然后切换到相应的 TaskManager/JobManager 页面（分析器选项卡）。
```

Comment:
```suggestion
- 首先，你应找出性能瓶颈的 TaskManager/JobManager，然后切换到相应的 TaskManager/JobManager 页面（分析器选项卡）。
```
and replace all others. 
Reference：https://cwiki.apache.org/confluence/display/FLINK/Flink+Translation+Specifications